src/test/java/com/example/glovo/test/OrderServiceTest.java

```diff
@@ -0,0 +1,109 @@
+package com.example.glovo.test;
+
+import com.example.glovo.dto.OrderDto;
+import com.example.glovo.dto.ProductDto;
+import com.example.glovo.entity.Order;
+import com.example.glovo.entity.Product;
+import com.example.glovo.exeption.OrderNotFoundExeption;
+import com.example.glovo.repository.OrderRepository;
+import com.example.glovo.repository.ProductRepository;
+import com.example.glovo.service.OrderService;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyInt;
+import static org.mockito.Mockito.when;
+
+class OrderServiceTest {
+
+    private OrderRepository orderRepository;
+    private ProductRepository productRepository;
+    private OrderService orderService;
+
+    @BeforeEach
+    public void prepare() {
+        this.orderRepository = Mockito.mock(OrderRepository.class);
+        this.productRepository = Mockito.mock(ProductRepository.class);
+
+        this.orderService = new OrderService(orderRepository, productRepository);
+    }
+
+    @Test
+    void ordersEmptyIfNoOrderAdded() {
+        var orders = orderService.getAll();
+        assertTrue(orders.isEmpty(), "Orders list should be empty");
+    }
+
+    @Test
+    void getAll() {
```

Comment:
according to the naming convention, the name of test case need to be getAll -> getAllTest